### DISCUSSION TITLE
how many Thailand excess deaths are due covid, suicide or other causes?

### DISCUSSION THREAD

**Discussion by djay** [djay/covidthailand#244] [Q&A]
Excess deaths are during peak covid deaths are significantly more than official covid deaths. How many are due to covid?
There are claims that the economic impact of the restrictions and loss of tourism and general economic downturn has increased the already high thailand suicide rate and this impact is significant compared to deaths by covid.

**Comment by djay** [comment]
# Excess deaths
The number of deaths in addition to what would be expected without the pandemic is a little variable since the offically recorded deaths in the 5 years before covid wasn't consistent, particularly in Aprils and 2019 and 2016, However even with those outliers the deaths are significantly more

![](https://practical-ritchie-cca141.netlify.app/outputs/deaths_excess_covid_5y_all.png)

Most of these excess deaths are in the elderly
![](https://practical-ritchie-cca141.netlify.app/outputs/deaths_excess_age_bar_5y_all.png)
![](https://practical-ritchie-cca141.netlify.app/outputs/deaths_pscore_age_all.png)

And not all in the same parts of the country
![](https://practical-ritchie-cca141.netlify.app/outputs/deaths_pscore_region_all.png)

Other estimates of covid deaths are
- [The economist coronavirus excess deaths tracker](https://www.economist.com/graphic-detail/coronavirus-excess-deaths-tracker)

# Estimated covid deaths

> The excess deaths directly caused by the COVID-19 infections accounted for approximately 75.0% of the all-cause excess deaths. 

[Estimation of excess all-cause mortality due to COVID-19 in Thailand - 2021-01-07](https://www.medrxiv.org/content/10.1101/2022.01.07.22268886v1)

> For Thailand, as of the end of 2021, the country reported a total of 21,700 Covid-related deaths. The analysis estimated around 35,200 excess deaths in Thailand from January 2020 to the end of 2021.

Source: [Lancet via The Thaiger, March 14, 2022](https://thethaiger.com/coronavirus/deaths-due-to-covid-19-pandemic-much-higher-than-reported-figures-study-shows)

[IHME forecast model](https://www.healthdata.org/covid/updates) uses an estimate of total deaths due to covid of 1.3 (April 2022) however their estimate has changed over time

# Increased deaths due to suicide

Early indications where that suicide rates were up by 22% - https://www.bangkokpost.com/thailand/general/1983271/suicide-rate-surges-22-

Later estimates were 5.9x prepandemic rates [Bangkok Post 6 MAR 2022](https://www.bangkokpost.com/thailand/general/2274423/3-million-infections-since-2020)
> It added that the suicide rate is now 5.9 higher than the rate prior to the pandemic.
> Dr Amporn said the rise in mental health issues is also attributed to the adverse impact of the spread of the Omicron variant.

[Worldbank](https://data.worldbank.org/indicator/SH.STA.SUIC.P5?locations=TH) puts the prepandemic suicide rate at 8.8 per 100k. It's not clear what rate 5.9x was based on however but if we assume worldbanks estimate then its 52 per 100k over a year or 0.14 per 100k per day.

During delta the covid death rate was much higher than this except in NE. During omicron it is comparable.
This is still not a fair comparison because the effect of the pandemic is long lasting so you can't just compare it to the death rate at the peaks. But also to estimate the number of covid deaths with no restrictions is also not easy.

![Deaths by region](https://github.com/djay/covidthailand/wiki/deaths_region_3.png)

But I think if it is 52 per 100k then this is a significant number

**Comment by istrik** [comment]
> Excess deaths are during peak covid deaths are significantly more than official covid deaths. How many are due to covid? There are claims that the economic impact of the restrictions and loss of tourism and general economic downturn has increased the already high thailand suicide rate and this impact is significant compared to deaths by covid.

I was wondering if the high number of dengue infections or other diseases, because of covid restrictions were very low in these last few years,  can be another cause to explain excess deaths and if there is a way to disentangle those effects

**Reply by djay** [reply]
There aren't many deaths due to dengue and certainly less during covid due to the less travelling and restrictions. https://www.thaipbsworld.com/public-advised-to-be-aware-of-dengue-fever-re-emergence-in-thailand/. 
Some excess deaths will be due to reluctance to go to hospitals so cancers that weren't detected soon enough, delayed operations etc. But most countries that investigate the cause of excess deaths normally come out with a figure around 70% due to covid. I think there was at least one study for this in thailand. (https://www.medrxiv.org/content/10.1101/2022.01.07.22268886v1.full 75%)

**Comment by istrik** [comment]
Hi Dylan
thanks so much for taking some time to answer to my comment. I appreciate
it.
It makes totally sense that one of the reasons could be to the reluctance
to go to hospital,  I actually know quite a lot of people who did that
mistake.

Btw I'm really impressed by the work you did with the dashboard about covid
in Thailand. I was so impressed that I started studying Python.
I've a few books for beginners and I was wondering if you could give me an
advice about a book you consider good to learn python.

I was hoping to come to one of the meetup in Bangkok but I suddenly moved
in Laos with my family. I hope to have the chance in future

All the best
Luciano

**Comment by djay** [comment]
it makes me happy to hear that. Good luck with your learning. 
Hybrid ThaiPy meetups should return next month so you can join online. There have been some attempts to get a community going in Laos. Encourage them to do so or start your own meetup. The good thing about meetups is you don't have to know the thing, just have the long term motivation to bring those people together. 
I really don't know enough to recommend one book over another. I created this a long time ago but can't say it's well researched - https://github.com/bkkhack/resources/wiki/Python. https://www.pythondiscord.com/ does seem to be a well run resource for getting help.

**Reply by djay** [reply]
There is a lot more rumours lately that the continued high excess deaths are caused by vaccines. 

Here is a very good post showing all the data against this theory. 

https://yourlocalepidemiologist.substack.com/p/covid-19-vaccines-and-sudden-deaths
